ConsecutiveSum.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Task: Consecutive Number Summer
+ * Description: Write a Java program that finds whether a given number can be expressed 
+ * as the sum of two or more consecutive natural numbers. If possible, the program should 
+ * print all possible combinations of consecutive natural numbers that sum up to the given number.
+ * 
+ * Example:

```

Comment:
don't include examples or other comments which are not important, on include task-name, description , name , date